romeo/asset-contract/tests/clarity-bitcoin-mini-deploy.test.ts

```diff
@@ -0,0 +1,21 @@
+import { beforeEach, describe, expect, it } from "vitest";
+
+beforeEach(async () => {
+  await vm.initSession(process.cwd(), "./Clarinet.toml");
+});
```

Comment:
```suggestion
```

This is now the default behavior of the test framework, so it's not needed anymore.

And this it what appears to make the test fail right now.

cc @friedger